src/main/java/org/broadinstitute/hellbender/tools/spark/sv/evidence/ReadMetadata.java

```diff
@@ -329,6 +330,18 @@ public static void writeMetadata(final ReadMetadata readMetadata,
                 writer.write(idx + "\t" + bounds.firstContigID + "\t" + bounds.getFirstStart() + "\t" +
                         bounds.getLastContigID() + "\t" + bounds.getLastStart() + "\n");
             }
+            writer.write("contigs map:\n");
+            readMetadata.contigNameToID
```

Comment:
You could just use the contigIDToName array instead of the map.  It's already in the right order.

Reply:
Ah, I didn't notice that member of ReadMetadata. I'll change it.